package.json

```diff
@@ -18,7 +18,7 @@
     "build:checkout": "yarn --cwd feature-libs/checkout run build:schematics && ng build checkout --configuration production",
     "build:digital-payments": "yarn --cwd integration-libs/digital-payments run build:schematics && ng build digital-payments --configuration production",
     "build:order": "yarn --cwd feature-libs/order run build:schematics && ng build order --configuration production",
-    "build:libs": "ng build core --configuration production && ng build storefrontlib --configuration production && yarn build:schematics && yarn build:user && yarn build:cart && yarn build:order && yarn build:checkout && yarn build:asm && concurrently --kill-others-on-fail -m 4 yarn:build:organization yarn:build:product yarn:build:product-configurator yarn:build:tracking yarn:build:storefinder yarn:build:smartedit yarn:build:qualtrics yarn:build:assets yarn:build:incubator yarn:build:cdc yarn:build:cds yarn:build:setup yarn:build:digital-payments",
+    "build:libs": "ng build core --configuration production && ng build storefrontlib --configuration production && yarn build:schematics && yarn build:user && yarn build:cart && yarn build:order && yarn build:checkout && yarn build:asm && yarn build:tracking  && concurrently --kill-others-on-fail -m 4 yarn:build:organization yarn:build:product yarn:build:product-configurator yarn:build:storefinder yarn:build:smartedit yarn:build:qualtrics yarn:build:assets yarn:build:incubator yarn:build:cdc yarn:build:cds yarn:build:digital-payments && yarn build:setup",
```

Comment:
Let's just use the line from the cart.
But I'm wondering, don't we need `:` after `yarn` for the setup lib?

Reply:
yup yup confused on why too from last bullet point [here](https://sap-cx.slack.com/archives/D02A0NHD3C0/p1643268177004900)

I think setup is fine since I guess they wanted to run it last after all the previous concurrent builds are done
But I don't understand why we need this change though 🤔  as it's been good to everyone so far.

Literally, setup went to the last part and tracking moved out from concurrent build to a blocking call